sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorAmqpConnection.java

```diff
@@ -69,24 +74,41 @@ public ServiceBusReactorAmqpConnection(String connectionId, ConnectionOptions co
         this.tokenManagerProvider = tokenManagerProvider;
         this.retryOptions = connectionOptions.getRetry();
         this.messageSerializer = messageSerializer;
-        this.managementChannelMono = getReactorConnection().then(
-            Mono.fromCallable(() -> {
-                return (ServiceBusManagementNode) new ManagementChannel(
-                    createRequestResponseChannel(MANAGEMENT_SESSION_NAME, MANAGEMENT_LINK_NAME, MANAGEMENT_ADDRESS),
-                    connectionOptions.getEntityPath(), connectionOptions.getTokenCredential(),
-                    this.tokenManagerProvider, this.messageSerializer, connectionOptions.getScheduler());
-            }))
-            .cache();
+        this.tokenCredential = connectionOptions.getTokenCredential();
+        this.scheduler = connectionOptions.getScheduler();
     }
 
     @Override
-    public Mono<ServiceBusManagementNode> getManagementNode() {
+    public Mono<ServiceBusManagementNode> getManagementNode(String entityPath) {
```

Comment:
Can the same `ServiceBusReactorAmqpConnection` get management nodes for different queues or topics?